_includes/v22.2/ui/active-statement-executions.md

```diff
@@ -8,7 +8,7 @@ Column | Description
 Statement Execution ID | The execution ID of the statement execution.
 Statement Execution | The SQL statement that was executed.
 Status | The status of the execution: `Preparing`, `Waiting`, or `Executing`.
-Start Time | The timestamp the execution started.
+Start Time (UTC) | A timestamp started by the execution.
```

Comment:
This has a different meaning now than previously. Isn't the value simply a timestamp that represents when the execution started?

Reply:
That was a mistake that I later corrected. The current line says:  `The timestamp when the execution started.